README.md

```diff
@@ -6,3 +6,49 @@ Steinburg VST SDK are in the 'shared' folder
 
 On Mac set an alias in ~/.zshrc for 'projucer' pointing to the projucer executable.
 E.g: `alias projucer=~/Documents/Personal/JUCE/Projucer.app/Contents/MacOS/Projucer`
+
+
+### Making a new plugin
+The boilerplate code for a new plugin is handled for you with a python script.
+```bash
+--> py makeSpectral.py -h
+usage: makeSpectral.py [-h] -p PATH -n NAME [-d DESCRIPTION] [-v {True,False}]
+
+Automate SpectralSuite boilerplate for new plugin.
+
+options:
+  -h, --help            show this help message and exit
+  -p PATH, --path PATH  The relative or absolute path to the directory containing new plugin
+  -n NAME, --name NAME  The name of the new plugin
+  -d DESCRIPTION, --description DESCRIPTION
+                        The description of the new plugin
+  -v {True,False}, --verbose {True,False}
+```
+```bash
+--> py makeSpectral.py -p NewThing/ -n "NEWNAME" -d "NEWDESC" -v True
+Populated NewThing/ with boilerplate files
```

Comment:
Maybe we don't need all of the output in the README file? Just the initial command and the ls+output below? 
I do like the output of the verbose option though. Very useful :-)

Reply:
### Yeah you're absolutely right.
How about this? Short and sweet!

---------
...[README]...
The boilerplate code for a new plugin is handled for you with a python script.
```bash
--> py makeSpectral.py -h
usage: makeSpectral.py [-h] -p PATH -n NAME [-d DESCRIPTION] [-v {True,False}]

Automate SpectralSuite boilerplate for new plugin.

options:
  -h, --help            show this help message and exit
  -p PATH, --path PATH  The relative or absolute path to the directory containing new plugin
  -n NAME, --name NAME  The name of the new plugin
  -d DESCRIPTION, --description DESCRIPTION
                        The description of the new plugin
  -v {True,False}, --verbose {True,False}
```
```bash
--> py makeSpectral.py -p NewThing/ -n "NEWNAME" -d "NEWDESC" -v True
--> ls NewThing/
NEWNAME.jucer		NEWNAMEParameters.cpp	NEWNAMEProcessor.h
NEWNAMEAudioPlugin.cpp	NEWNAMEParameters.h	UiContainer.cpp
NEWNAMEAudioPlugin.h	NEWNAMEProcessor.cpp	UiContainer.h
```

Reply:
Yea that's great and much easier to read too. Thanks 👍